tests/cpp/operator/mkldnn_operator_test.cc

```diff
@@ -523,8 +575,8 @@ void TestOpEx(const OpAttrs &forward_attrs, const OpAttrs &backwards_attrs) {
     for (int i1 = 0; i1 < in_arrs.size(); i1++) {
       auto in_arr = in_arrs[i1];
 
-      // TODO(alex): (MXNET-845) Remove when MKLDNN supports other dims
-      if (in_arr.arr.shape().ndim() != 4)
+      if (forward_attrs.accept_dims.find(in_arr.arr.shape().ndim()) ==
```

Comment:
we should check for accept_dims not empty here.

Reply:
If accept_dims is empty, the "if" condition results to true, and it simply skips the test.

Reply:
dont we want all tests which use TestOpEx to atleast provide one element in accept_dims. Other option is to have a default ndim like 4 inside accept_dims.

Reply:
Added a check for non-empty accept_dims.